machine/accton/accton_as5916_54x/busybox/patches/series

```diff
@@ -0,0 +1 @@
+onie-syseeprom-write-2b-delay-4000.patch
```

Comment:
maybe not for this platform, but I see this type of patch so often.....  perhaps we should make a busybox CONFIG_SYS_EEPROM_DELAY parameter that platforms can put in busybox/conf/config...